test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,54 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+
+    Scenario: Service Binding Request without application selector and with Backend DB
+        Given Backend infrastructure is running
+        * Backend CR "backend-demo" is applied
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+                annotations:
+                    servicebindingoperator.redhat.io/status.ready: 'binding:env:attribute'
+                    servicebindingoperator.redhat.io/spec.host: 'binding:env:attribute'
+            spec:
+                host: example.common
+            """
+        * Service Binding request "binding-request-backend" is applied
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: binding-request-backend
+            spec:
+                backingServiceSelector:
+                    group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    resourceRef: backend-demo
+                    id: SBR
+                customEnvVar:
+                  - name: CustomReady
+                    value: '{{ .SBR.status.ready }}'
+                  - name: CustomHost
+                    value: '{{ .SBR.spec.host }}'
+            """
+        When Backend status in "backend-demo" is updated
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-demo
+                annotations:
+                    servicebindingoperator.redhat.io/status.ready: 'binding:env:attribute'
+                    servicebindingoperator.redhat.io/spec.host: 'binding:env:attribute'
+            spec:
+                host: example.common
+            status:
+                ready: true
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-backend" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-backend" should be changed to "False"
+        And Secret "binding-request-backend" contains CustomReady custom env name with value true
```

Comment:
To be sure that the value really got updated/propagated after service status update, we should also have this check after line 253. The assertion there will be slightly different because the value either does not exist or has some other content because status part was different.

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/2899fd0b882ae30ee3b19d149d9329703c878e0c

Reply:
Updated and done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/b904a2186b93b06ef06213f2a23139572628abdc